lib/user.rb

```diff
@@ -0,0 +1,12 @@
+class User
+  attr_reader :id
+  attr_accessor :username, :password, :first_name, :last_name, :admin
+
+  def initialize(username, password, first_name, last_name = nil, admin = false)
+    @username= username
+    @password= password
+    @first_name= first_name
+    @last_name= last_name.empty? ? nil : last_name # Last name can be left epmty
```

Comment:
what if we don't handle this case? This would be anyways nil, right?

Reply:
Yes the default value taken for last_name in the constructor is nil, this is used in case user does not want to input last_name they can just press enter to skip it